app/assets/javascripts/datatables.coffee

```diff
@@ -0,0 +1,24 @@
+$(document).on 'turbolinks:load', ->
+  tableElementIds = [
+    '#passengers.admin-table'
+    '#passengers.dispatch-table'
+  ]
+  i = 0
+  while i < tableElementIds.length
+    tableElementId = tableElementIds[i]
+    if $.isEmptyObject($.find(tableElementId))
+      i++
+      continue
+    table = undefined
+    if $.fn.DataTable.isDataTable(tableElementId)
+      table = $(tableElementId).DataTable()
+    else
+      table = $(tableElementId).DataTable() #Some options would go here...
```

Comment:
Oh, would they?  What sort of options?

Reply:
I think the coffeescript can theoretically handle what datatables.js does (choosing whether the admin table or dispatch table shows up). But ... well the js is handling that nicely right now.

Reply:
Here's the weird thing ... If I get rid of my `else` statement but leave the `if`, the duplicate headers/footers error returns. So that means ... `DataTable.isDataTable(tableElementId)` is false? (Or at least, not true?) It's not a DataTable tableElementId? I'm trying to figure out how to parse this and what the implications are ...